example/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	_ "embed"
+	"fmt"
+	"image"
+	"os"
+	"path/filepath"
+
+	xgb "github.com/Elvenson/xgboost-go"
+	"github.com/Elvenson/xgboost-go/activation"
+	"github.com/viamrobotics/ocean-prefilter/oceanprefilter"
+)
+
+//go:embed xg_boost_dump.json
+var modelbytes []byte
```

Comment:
I'm not sure if the bytes should be only in the example directory-- you probably want it in the main package

Reply:
I have it in both! Because in the test functions you have to setup your runconfig yourself

Reply:
my mistake! didn't see it in both